### PR TITLE
Task/#leip 202 android upload does not work

### PR DESCRIPTION
**Pull request by muthenberg**

### CODE REVIEW

File: src/test/resources/example-image-enterprise.jpg
Comment (hb0) [inline src/test/resources/example-image-enterprise.jpg:1]: Is this file used anywhere? I only found the usage of example-image.png in the test code.

**Review by hb0 — CHANGES_REQUESTED** [review]

File: src/main/kotlin/de/cyface/collector/storage/cloud/GoogleCloudStorageService.kt [hunk 1]
```diff
@@ -182,15 +192,20 @@ class CloudStorageSubscriber<in T : Buffer>(
...
     override fun onNext(t: T) {
         // Write to Google Cloud
+        logger.debug("Streaming ${t.bytes.size} Bytes to Google Cloud Storage.")
+        streamedBytes += t.bytes.size
         cloudStorage.write(t.bytes)
-        this.subscription.request(1)
+        this.subscription.request(8192)
+        logger.debug("Progress: ${streamedBytes.toDouble()/totalBytes.toDouble() * 100.0} %")
     }
 }
```
Comment (hb0) [inline src/main/kotlin/de/cyface/collector/storage/cloud/GoogleCloudStorageService.kt]: So was this the actual bug to be fixed? Or did I miss the actual fix somewhere else

Comment (hb0) [reply src/main/kotlin/de/cyface/collector/storage/cloud/GoogleCloudStorageService.kt]: Also: CI fails

Comment (muthenberg) [reply src/main/kotlin/de/cyface/collector/storage/cloud/GoogleCloudStorageService.kt]: > So was this the actual bug to be fixed? Or did I miss the actual fix somewhere else

I guess so.

File: src/test/resources/example-image-enterprise.jpg
Comment (muthenberg) [reply src/test/resources/example-image-enterprise.jpg:1]: It is used if I need a smaller image.

File: src/test/kotlin/de/cyface/collector/storage/cloud/UploadPictureIT.kt [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+            .put("db_name", "cyface")
+            .put("connection_string", "mongodb://localhost:27019")
+            .put("data_source_name", "cyface")
+        val uploadExpiration = 60000L
+        val measurementPayloadLimit = 104857600L
+        val storageType = GoogleCloudStorageType(
+            collectionName = "cyface",
+            projectIdentifier = "object-storage-412713",
+            bucketName = "cyface",
+            credentialsFile = "/Users/muthmann/.config/gcloud/application_default_credentials.json"
```
[the remark this answers is not in the document]
Comment (muthenberg) [reply src/test/kotlin/de/cyface/collector/storage/cloud/UploadPictureIT.kt]: This should not be here.

**Review by hb0 — APPROVED** [review]

**Pull request merged**